src/plugins/opensearch_dashboards_react/public/overview_page/overview_page_header/overview_page_header.tsx

```diff
@@ -47,24 +47,28 @@ import './index.scss';
 
 interface Props {
   hideToolbar?: boolean;
+  /** @deprecated use showIcon */
   iconType?: IconType;
```

Comment:
Is `iconType` still honored if we're just deprecating it?

Reply:
It never was.